server/src/test/java/io/druid/client/CachingClusteredClientTest.java

```diff
@@ -2363,4 +2363,120 @@ public void testTimeBoundaryCachingWhenTimeIsInteger() throws Exception
         makeTimeBoundaryResult(new DateTime("1970-01-05T01"), new DateTime("1970-01-05T01"), null)
     );
   }
+
+  @Test
+  public void testGroupByCachingRenamedAggs() throws Exception
+  {
+    GroupByQuery.Builder builder = new GroupByQuery.Builder()
+        .setDataSource(DATA_SOURCE)
+        .setQuerySegmentSpec(SEG_SPEC)
+        .setDimFilter(DIM_FILTER)
+        .setGranularity(GRANULARITY)
+        .setDimensions(Arrays.<DimensionSpec>asList(new DefaultDimensionSpec("a", "a")))
```

Comment:
can we test with a different output name as well? It's unclear which one should be stored in the cache